wpimath/src/main/java/edu/wpi/first/math/geometry/Rotation2d.java

```diff
@@ -248,25 +240,22 @@ public Angle getMeasure() {
   }
 
   /**
-   * Returns the radian value of the Rotation2d.
+   * Returns the radian value of the Rotation2d constrained within (-π, π].
```

Comment:
```suggestion
   * Returns the radian value of the Rotation2d constrained within [-π, π].
```
`getRadians()` can produce -π if m_sin is -0 (and m_cos is negative). For example, `Rotation2d.kPi.unaryMinus().getRadians()` produces -π, as does `Rotation2d.fromRadians(-Math.PI).getRadians()` and `new Rotation2d(-Math.PI).getRadians()`.
This probably also applies to C++, but I can't test at the moment. [`gcem::atan2`](https://gcem.readthedocs.io/en/latest/api/trigonometric_functions.html#atan2-func-ref) also returns `-π` if y is -0 (and x is negative), but it's harder for me to figure out when C++ can produce -0.